### ISSUE TITLE
[BUG]: Some questions about exp6

### ISSUE THREAD

**Issue by ViKeei** [Unity-Technologies/UnityRenderStreaming#868]
### Package version

3.1.0-exp.6

### Environment

```markdown
* OS:Windows11
* Unity version: Unity 2023.1b3、Unity 2021.3.2
* Graphics API:DX12
* Browser: Edge dev 111.0.1660.9
```

### Steps To Reproduce

**Question 1：**
After fixing InputSystem Background Behavior and InputSystem PlayMode Input Behavior in the Render Streaming Wizard window, restarting Unity Settings will become invalid. Manually creating an Input System Setting asset and clicking Fix also causes the above problems unless the Settings are manually created and manually changed.
**Question 2：**
After I set up a new scene according to create-scene.md, I run the game and adjust the Streaming Size, Video Codec, Framerate, Bitrate, Scale Resolution Down and other parameters in the Video Stream Sender, but the web page does not change. For example, the Straming Size at the bottom of the page is 1920*1080, but the Video Stream Sender is set to 1280*720, and the whole page is still, the dynamic objects in the page such as clouds and water do not move unless Run On Awake is turned off. I have tested exp5 in the same project without the above problem. I have tested it in Unity2021 and Unity2023.
Is there something wrong with the way I'm doing it?

### Anything else?

[Video_20230211180714_edit.zip](https://github.com/Unity-Technologies/UnityRenderStreaming/files/10713303/Video_20230211180714_edit.zip)

**Comment by karasusan** [comment]
@ViKeei
Can you check the Render Streaming settings in Project Settings Window?
It looks the automatic streaming is enabled.

You can see the step to turn the flag off on this document.
https://github.com/Unity-Technologies/UnityRenderStreaming/blob/main/com.unity.renderstreaming/Documentation~/create-scene.md

**Comment by kannan-xiao4** [comment]
@ViKeei 
About Question1, I can reproduce that issue and fix on #870.

**Comment by ViKeei** [comment]
@karasusan I checked the Settings carefully and fixed the problem, thank you！:D
